!--#include virtual="/inc/page-header.html"-->
<!--#include virtual="/inc/page-layout-top-wide.html"-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
    <head>
        <title>ListServ Archive: Gene Expression in Dictyostelium</title>
    </head>
    <body>
        <p>
            <a name="top" id="top"></a> <a name="geneex" id="geneex"></a>
        </p>
        <h1>
            <a name="geneex" id="geneex"><b>Gene Expression in Dictyostelium</b></a>
        </h1>
        <hr>
        <ul>
            <li>
                <a href="listserv_archive_geneex.html#geneex19">Is there an appropriate Tet On or Tet Off expression system for Dictyostelium?</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
        </ul>
        <p>
            <br>
            <a href="listserv_archive_geneex.html#geneex_exsys"><b><font color="black">Dictyostelium as an Expression System</font></b></a>
        </p>
        <ul>
            <li>
                <a href="listserv_archive_geneex.html#geneex16">I would like to know if Dictyostelium would be a suitable expression system to obtain a fully glycosylated product.</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
        </ul>
        <p>
            <a href="listserv_archive_geneex.html#geneex_reporter"><b><font color="black">Reporter Genes</font></b></a>
        </p>
        <ul>
            <li>
                <a href="listserv_archive_geneex.html#geneex18">I would like to express a GFP-tagged version of a protein under its own promoter.</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_geneex.html#geneex18"></a><br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex1">Does anyone know the half life of GFP expressed in Dictyostelium?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_geneex.html#geneex1"></a><br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex2">Has anyone tried expressing the EGFP form of GFP from Clontech?</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex3">Does the codon usage of the GFP gene (some of the different types of GFP have been constructed using the human codon usage tables) need to be changed for significant/detectable expression of GFP in Dictyostelium?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_geneex.html#geneex3"></a><br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex4">Has anyone used any of the coloured variants of GFP in Dicty?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_geneex.html#geneex4"></a><br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex15">RFP-ABD</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex5">Has anyone yet tried hrGFP in Dicty?</a>
            </li>
        </ul>
        <p>
            <br>
            <a href="listserv_archive_geneex.html#geneex_tags"><b><font color="black">Protein Tags</font></b></a>
        </p>
        <ul>
            <li>
                <a href="listserv_archive_geneex.html#geneex17">Can you suggest a good protein tag for Co-IP in Dicty?</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex6">I was wondering if there was any consensus on which protein tags work best with Dicty proteins.</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex7">Is anyone having trouble using myc as an epitope tag? We have tried placing it on two proteins so far and are having no success.</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex8">Are there any experiences whether N-terminal FLAG-epitopes fused to a protein of interest work well in Dictyostelium and whether FLAG-antibodies can be used for indirect immunofluorescence experiments with regard to background fluorescence?</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex9">Has anybody used the FLAG epitope in Dicty? Any information on positioning of the epitope, cross reactivity of the antibodies etc would be greatly appreciated.</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_geneex.html#geneex10">My lab has recently had bad luck with the myc epitope not being visible (by western) when fused at the amino term. of proteins in Dicty. We plan to try an HA tag, and would appreciate any info that anyone has (successes, failures , problems, or other general comments!!!)</a>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex19" id="geneex19"></a> <b>Is there an appropriate Tet On or Tet Off expression system for Dictyostelium, and if yes, where can I purchase it from? Best regards<br></b> <i>Can ÜNAL, Institute for Molecular Infectionbiology , Wuerzburg/Germany, March 15, 2006</i><br>
        </p>
        <ul>
            <li>Both tetON and tetOFF systems exist for Dicty. The tetOFF system was pioneered by Dingermann and colleagues, but the most frequently used vectors are those constructed by <a href="/db/cgi-bin/dictyBase/reference/reference.pl?refNo=387">Blaauw et al 2000</a>. The Dicty tetON system is derived from yeast vectors described in <a target="new" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=10859354&amp;query_hl=15&amp;itool=pubmed_docsum">Urlinger et al 2000</a>; the rtTa-M2 element from was recloned into the Blaauw vector MB35 by Adriano Ceccarelli, and students in my lab have shown that the performance of this tetON system in Dicty (induction factor, response time) is similar to that of the tetOFF system.<br>
                Both the original tetOFF MB35 and its tetON version are neo vectors. In practice one needs a Dicty strain carrying one or the other of these, which one overtransforms with a second, blasticidin vector (Blauuw et al MB38) carrying the gene of interest. Once one has cloned one's gene into MB38, it can be overtransformed into both tetON and tetOFF strains. Both MB35 and MB38 are in the stock center, as are cells containing MB35. The tetON version of MB35 will go to the stock center today.<br>
                I and others have also made constructs in which both the tet-binding protein and the gene it controls are combined in a single neo vector. These constructs are helpful in particular applications but may suffer from elevated basal (noninduced) activity.<br>
                <i>-Harry MacWilliams, Biologiezentrum der Ludwig-Maximilians-Universität</i>
            </li>
        </ul>
        <hr>
        <p>
            <a name="geneex_exsys" id="geneex_exsys"></a>
        </p>
        <h2>
            <a name="geneex_exsys" id="geneex_exsys"><b>Dictyostelium as an Expression System</b></a>
        </h2>
        <hr>
        <p>
            <a name="geneex16" id="geneex16"></a> <b>I would like to know if Dictyostelium would be a suitable expression system to obtain a fully glycosylated product. I am interested in a special class of plant proteins rich in arabinose and galactose.</b><br>
            <i>-Christophe Reuzeau, 19, 1998</i><br>
        </p>
        <ul>
            <li>In general secreted Dicty glycoproteins have little galactose and to my knowledge no arabinose. Of course this may mean that the right glycoproteins have not yet been characterised There is a recent report of a Dicty nuclear protein containing an unusual galactose/fucose/glucosamine structure (<a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=6257">Teng-umnuay et al</a>).<br>
                Dicty N-linked glycosylation are similar the mammalian high mannose structures, with no lactosamine or sialic acids as in complex mammalian glycosylation, nor is there any polymannose extensions as in yeast. In O-linked glycosylation dicty substitutes N-acetyl glucosamine for GalNac; this glycoslytion is also reported in other protozoa eg Faliciparum. Dicty also has several phosphodiester linked forms of O-linked glycosylation; this type of glycosylation is found in pathogenic protozoa eg Leshmania.<br>
                In recombinant, secreted proteins, Dicty glycosylates the same N-linked sites as mammlian cells, but only glycosylates some O-linked sites recognised in mammals, eg XPXX motif. Dicty does not appear to glycosylate sites that are not recognised by mammals, unlike yeast where hyperglycosylation of recombinant protiens is common.<br>
                I do not know of any plant glycoproteins that have been expressed in Dicty, but check the codon usage of your gene is not too dissimilar to Dicty.<br>
                <i>-Martin Slade, 3 Sep 2003</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex_reporter" id="geneex_reporter"></a>
        </p>
        <h2>
            <a name="geneex_reporter" id="geneex_reporter"><b>Reporter Genes</b></a>
        </h2>
        <hr>
        <p>
            <a name="geneex18" id="geneex18"></a> <b>I would like to express a GFP-tagged version of a protein under its own promoter.</b><br>
            <i>-Jennifer Lee, Vassar College, July 20, 2006</i><br>
        </p>
        <ul>
            <li>One approach would be to insert a GFP coding domain into the chromosomal gene using homolgous recombination. Sincerely,<br>
                <i>-Chris West, University of Oklahoma Health Sciences Center</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>One solution to your problem is the one that Arturo Delozanne's lab used for the lvsA gene (Gerald et al. Traffic, 2002). They did a targeted insertion of GFP into the lvsA locus. This is the only example I know of this approach, but it should work generally.<br>
                <i>-David Knecht, University of Connecticut</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Ralph Graf used the same approach to label the endogenous gamma tubulin gene - clever idea! see <a href="/db/cgi-bin/dictyBase/reference/reference.pl?refNo=1210">Daunderer C, Graf RO 2002</a><br>
                <i>-Michael Koonce, Wadsworth Center</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>If you are thinking of using a reporter gene, then you might consider the unstable beta-galactosidase, which has worked very well in my hands. This enables one to measure current promoter activity, whereas GFP or GAL (or even Luciferase, in Dicty) have relatively long halflives, and give accumulated activity measurements. Please contact me if you need this reporter. best wishes,<br>
                <i>-Harry MacWilliams, Ludwig-Maximilians-Universität München</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex1" id="geneex1"></a> <b>Does anyone know the half life of GFP expressed in Dictyostelium?</b><br>
            <i>-Anne Hitt, Oakland University, Rochester, MI, 7 Jun 1997</i><br>
        </p>
        <ul>
            <li>It is very stable in Dictyostelium. In many cases too stable. GFP is very resistant to photobleaching. As many pointed out, this can be good or bad depending on the experiment. Also, it can be fixed with cold methanol/formaldehyde.
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex2" id="geneex2"></a> <b>Has anyone tried expressing the EGFP form of GFP from Clontech?</b><br>
            <i>Anne Hitt, Oakland University, Rochester, MI, 7 Jun 1997</i><br>
        </p>
        <ul>
            <li>Wild type GFP works OK in Dictyostelium. The S65T variant is reported to work better.<br>
                <br>
            </li>
            <li>I have used EGFP-C1 from Clontech to fuse with the myosin regulatory light chain (RLC) and expressed the EGFP-RLC fusion with Actin 15 promoter (EGFP 5' to RLC). The outcome is that this construct failed to even express in Dicty. I have since remade the fusion construct with S65TGFP (original non-humanized codon) and have just found out that the express very well in Dicty. The possible explanations for the EGFP result are: Dicty doesn't like human codon and/or the 5'UT of the EGFP-C1 construct unfit for Dicty expression (too GC rich). I have corrected both problem in the S65TGFP-RLC fusion construct and seemed to be working.<br>
                <i>-Tung-Ling Chen, Northwestern University, Chicago, IL</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex3" id="geneex3"></a><b>Does the codon usage of the GFP gene (some of the different types of GFP have been constructed using the human codon usage tables) need to be changed for significant/detectable expression of GFP in Dictyostelium?</b><br>
            <i>-Anne Hitt, Oakland University, Rochester, MI, 7 Jun 1997</i><br>
        </p>
        <ul>
            <li>Codon usage does not seem to be a problem for expression.<br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <b>Are there any other peculiarities about the expression of GFP in Dictyostelium that one should be aware of?</b><br>
            <i>-Anne Hitt, Oakland University, Rochester, MI, 7 Jun 1997</i><br>
        </p>
        <ul>
            <li>Although generally non-toxic, GFP can be toxic if driven by a strong promoter.<br>
                <br>
            </li>
            <li>GFP is very resistant to photobleaching, but Dictyostelium are very sensitive to strong light/heat. Work arounds suggested include using a good low light camera (first choice) or possibly a heat mirror (~$100).<br>
                <br>
            </li>
            <li>I see that wild-type GFP is stable, but I have two N-terminally modified GFPs with shorter halflives. For one the T1/2 is "about six hours" (good measurements are difficult); for the other, circa 20 minutes. The expression pattern seen with labile GFP resembles that of labile gal in the one case (ecm0) that I have examined carefully.<br>
                <i>-Harry MacWilliams, Ludwig-Maximilians-Universität, Muenchen, Germany</i><br>
                <br>
            </li>
            <li>A few good places to look for information include<br>
                <ul>
                    <li>
                        <a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=8664">Cubitt, A. B., Heim, R., Adams, S. R., Boyd, A. E., Gross, L. A., and Tsien, R. Y. (1995) Understanding, improving and using green fluorescent proteins. Trends Biochem. Sci. 20, 448-455.</a>
                    </li>
                    <li>
                        <a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=1706">Fey, P., Compton, K., and Cox, E. C. (1995). Green fluorescent protein production in the cellular slime molds, Polysphondelium palladium. and Dictyostelium discoideum. Gene 165, 127-130.</a>
                    </li>
                    <li>
                        <a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=2667">Hodgkinson, S. (1995) GFP in Dictyostelium. Trends Genet 11, 327-328.</a><br>
                    </li>
                    <li>
                        <a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?author=Prasher">Prasher DC (1995) Using GFP to see the light. Trends Genet 11, 320-323.</a><br>
                        &lt;//li&gt;
                    </li>
                    <li>
                        <a target="new" href="http://www.clontech.com/clontech/Manuals/GFP/Detection.html">www.clontech.com/clontech/Manuals/GFP/Detection.html</a>
                    </li>
                </ul>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex4" id="geneex4"></a> <b>Has anyone used any of the coloured variants of GFP in Dicty? And if yes, could you tell me if they were bright enough to see easily.</b><br>
            <i>-Richard Tuxworth, University of Minnesota, MN, 8 Sept 1999</i><br>
        </p>
        <ul>
            <li>We have used a blue version of GFP (Y66H) and, as you would expect, it has a much reduced signal relative to S65T. It is not very easy to visualize by regular epifluorescence, but easily detected by fluorimetry.<br>
                <i>-Adam Kuspa, Baylor College of Medicine, Houston, Texas, 9 Sept 1999</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex15" id="geneex15"></a> <b>Dear Dave, I have made a fusion of RFP-ABD, using your GFP-ABD as the template for PCR to amplify fragments with appropriate ends for inserting it between actin15 promoter-RFP and actin 6 terminator in pBIG (a derivative of pATNB43). Cells transformed with this plasmid are brightly fluorescent, but the patterns are somewhat different from that of GFP-ABD. I have an impression that the RFP-ABD is more cortical, and much less abundant in lamellipodia, than GFP-ABD. I am wondering if you have also made an RFP-ABD construct, and had similar experiences. Because the pattern of GFP is more reasonable than that of RFP, I suspect it is RFP-ABD which has a problem, if one of the two has a problem. I am wondering if RFP and GFP are somewhat different (I found out that RFP has more acidic side chains on the surface, some of which form clusters, as predicted from the alignment of the primary structures of GFP and RFP), or whether this difference is due to the fact that our RFP-ABD is on a multicopy extrachromosomal vector whereas your GFP-ABD is on an integration plasmid, or to some errors occurred during PCR amplification. I appreciate it very much if you could share your thoughts with me. With best reagards,</b><br>
            <i>-Taro Uyeda, National Institute for Advanced Interdisciplinary Research, Tsukuba, Japan, 23 Aug 2000</i><br>
        </p>
        <ul>
            <li>I think the problem maybe with the RFP itself. I made stable RFP expressing mammalian cell lines that tend to lose RFP expression from time to time. Some cells seem to concentrate the red in vesicles and get rid of them by exocytosis. I just had an e-mail correspondence with John Condeelis's technician who told me that RFP precipitates out more than GFP. So it maybe an inherent problem with the RFP and not the vector because I use the same vector with GFP and the cell lines are very stable.<br>
                <i>-Su Dharmawardhane, The University of Texas at Austin, TX, 23 Aug 2000</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>It seems that RFP has some problem based on what you told me before and on these emails. Margaret Clarke of Oklamoma said she heard about "bad" things about RFP, particularly "aggregation".<br>
                <i>-Yoshio Fukui, Northwestern University Medical School, Chicago, USA, 23 Aug 2000</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex5" id="geneex5"></a> <b>Has anyone yet tried hrGFP in Dicty? It might conceivably get over the variable expression of eGFP.</b><br>
            <i>-Robert Insall, 19 Jul 2001</i><br>
        </p>
        <ul>
            <li style="list-style: none">The summary of the hr-GFP story:<br>
                <br>
            </li>
            <li>hr-GFP is a GFP from another species of jellyfish which is said to be less toxic than normal GFP when expressed in mammalian cells.
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>There is fragmentary evidence that normal GFP is toxic in Dicty. Nothing to convince a sceptical reviewer either way. The most common observation is that cells - even from clonal transformants - express widely varying amounts of GFP even after cloning. Some people have also seen transformants lose GFP expression over time.
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Nobody appears to have tried hr-GFP to see if it behaves any differently. Someone ought to. I might.<br>
                <i>-Robert Insall, 8 Aug 2001</i><br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex_tags" id="geneex_tags"></a>
        </p>
        <h2>
            <a name="geneex_tags" id="geneex_tags"><b>Protein Tags</b></a>
        </h2>
        <hr>
        <p>
            <a name="geneex17" id="geneex17"></a> <b>Can you suggest a good protein tag for Co-IP in Dicty? I am trying to add a small peptide tag to a protein for Co-IP in Dicty. Now I am having a difficult time to choose what else to use. Our lab used His, Flag and Myc (single) tags before, but the background is too high. I also tried GFP on my protein, and the fusion caused loss of function.</b><br>
            <i>-Guokai Chen, Baylor College of Medicine, 20 Oct 2005</i><br>
        </p>
        <ul>
            <li>Try HA tag.<br>
                <i>-Chirag Mandavia, Hunter College, NY</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>You might look at the TAP tag literature and <a target="new" href="http://www-db.embl-heidelberg.de/jss/servlet/de.embl.bk.wwwTools.GroupLeftEMBL/ExternalInfo/seraphin/TAP.html">Seraphin's web site</a>.<br>
                <i>-Tex Cox, Princeton University, NJ</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Antibodies are expensive and nobody wants to mess around with them. As the first step, I think it would make a lot of dicty researchers life happier if somebody can compile a list of anti-tag antibodies that gives a clean background when probed to wild type AX4 or AX2 strains under normal western or IP conditions, and keep that list at DictyBase. Most antibodies are not made against native dicty proteins and are simply not acceptable even applied to dicty homologues. We can also add to the list other commonly used antibodies, such as the question just raised for tubulin, or antibodies to other subcellular marker proteins.<br>
                <i>-Patrick Zhang</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>We have made series of vectors for protein expression under actin15 in D.discoideum with TAP tags at either the N or C terminus either alone or in combination with a GFP or YFP tag. These vectors have proved succesful in pull down of members of the Arp2/3 complex which we used as a test system with p34-Arc as bait. In cases where pull down was not successful the bait protein itself comes back very pure after the dual affinity purification. You are welcome to use any of our vectors if you specify what you would like.<br>
                <i>-Pauline Schaap</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>You can search the <a target="new" href="/db/cgi-bin/dictyBase/SC/searchform.pl">Dicty Stock Center</a> for these vectors.<br>
                <i>-dictyBase</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
        </ul>
        <hr>
        <p>
            <a name="geneex6" id="geneex6"></a> <b>I need to place small tags on Dd proteins to facilitate IPs and detection, both in vivo and in vitro. I was wondering if there was any consensus on which protein tags work best with Dicty proteins.</b><br>
            <i>-Diana Caracino, Emory University, Atlanta, GA, 24 Feb 2003</i><br>
        </p>
        <ul>
            <li>Of the half dozen responses I received, most gave a very favorable response for Flag tags. C-myc came in second, and T7 was also mentioned. Hexa-his tags do not work well.<br>
                <i>-Diana Caracino, Emory University, Atlanta, GA, 26 Feb 03</i><br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex7" id="geneex7"></a> <b>Is anyone having trouble using myc as an epitope tag? We have tried placing it on two proteins so far and are having no success. In both cases we put it almost at the N-terminus (just after the Met or one amino acid further down). We used the 10 amino acid epitope recognized by the 9E10 monoclonal antibody. This is one "E" short of what Devreotes used, apparently successfully, on cAR1. Other data suggest the RNAs are being made but the protein is not, or is degraded very rapidly. Does this sound familiar to anyone? Does Dicty seem to have a problem with myc? Is the position, if you are going to put it on the N-terminus, critical? Or are we just unlucky with placing it there in the two cases we have tried. Before we invest time switching positions or epitopes, I would love to know of others successes and failures here.</b><br>
            <i>-Karl Saxe, 10 Jul 1997</i><br>
        </p>
        <ul>
            <li>We've been doing a lot with both c-myc and FLAG tags and in some cases double tags (myc on one and FLAG on the other). We usually place it at the very N terminus (inside the ATG of course) or the very C terminus. The myc tag needs the N residue at the end of it to work with the monoclonal. The sequence we use is: EQKLISEEDLN. We've put tags on both ends of proteins and in some cases it works and in others it does not. It's a bit of a crap shoot. Also, be sure to sequence and check the clones. Oligos can be bad and PCR can be a disaster. We've had all types of problems related to both PCR errors and bad oligos, but if you have a good expression clone, it works most of the time without a hitch. The myc tag is working well for Westerns and immunfl. (IPs not really tried), while the FLAG works great for IPs and Westerns (a bit too much background in our hands for immunofl. for many cellular proteins).<br>
                <i>-Rick Firtel, UCSD, CA, 10 Jul 1997</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Thanks alot. Had no clue about the N at the C-terminus of the epitope. It's not on the original human myc sequence as I understand it, but have heard people here mumble about the importance of the C-term without any details. That's a big help I suspect. Just for your info we have succeeded in using 9E10 to IP some in vitro translated myc-tagged protein but the efficiency was poor. Suggests if we had the N on there it might IP great.<br>
                <i>-Karl Saxe, 11 Jul 1997</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>From experience, that N at the C-terminus is essential.<br>
                <i>-Rick Firtel, UCSD, CA, 11 Jul 1997</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex8" id="geneex8"></a> <b>Are there any experiences whether N-terminal FLAG-epitopes fused to a protein of interest work well in Dictyostelium and whether FLAG-antibodies can be used for indirect immunofluorescence experiments with regard to background fluorescence?</b><br>
            <i>-Ralph Graef, Universitaet Muenchen, Germany, 25 Oct 2000</i><br>
        </p>
        <ul>
            <li>I have successfully used N-terminal FLAG fusions in Dicty. For immunoprecipitation, I use Sigma M2. IBI-Kodak goat anti-flag used to be the best in our hands, but it's out of market now. For immunofluorescence, I haven't had good luck with any of sigma monoclonals or other polyclonals.<br>
                <i>-Kim, Leung, LCDB/NIDDK/NIH, 25 Oct 2000</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>They work very well. Sometimes ones sees a slight amount of nuclear staining in control cells. We have and still use them extensively, along with HA and myc. All work very well.<br>
                <i>-Rick Firtel, UCSD, CA, 25 Oct 2000</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex9" id="geneex9"></a> <b>Has anybody used the FLAG epitope in Dicty? Any information on positioning of the epitope, cross reactivity of the antibodies etc would be greatly appreciated. Basically, I'm looking for a tag that can be used in conjunction with glutaraldehyde fixation and has no cross reactivity.</b><br>
            <i>-Andrew Wilkins, University College London, London, UK, 26 Sep 1997</i><br>
        </p>
        <ul>
            <li>We've used FLAG a lot at both ends of proteins. Both ends work well, but that does not mean that it works on all proteins. There are a few cross-reacting bands on Westerns but the results are good. On immunofl., there is some background nuclear staining. It's not bad but for lower signals, it makes the tag not functional for such studies. The myc tag does not have this problem. Our tendency now is to use myc, unless were need two tags on one protein (one on each end to examine processing or other questions) or if we need to put two different tagged proteins into a cell. For FLAG, there are antibodies specific for each type of tag. They seem to work as advertised.<br>
                <i>-Rick Firtel, UCSD, CA, 3 Sep 2003</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="geneex10" id="geneex10"></a> <b>My lab has recently had bad luck with the myc epitope not being visible (by western) when fused at the amino term. of proteins in Dicty. We plan to try an HA tag, and would appreciate any info that anyone has (successes, failures , problems, or other general comments!!!)</b><br>
            <i>-Tom Egelhoff, 14 Apr 1998</i><br>
        </p>
        <ul>
            <li>We also had a great deal of problems with myc at the N-terminal. We speculated that it was being cleaved. We have much better luck with HA remaining intact. However the antibody doesn't seem to be as specific.<br>
                <i>-Peter Deverotes, 14 Apr 1998</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p><!--#include virtual="/inc/page-layout-bottom.html"-->
        <!--#include virtual="/inc/page-footer.html"-->
    </body>
</html>
