<!--#include virtual="/inc/page-header.html"-->
<!--#include virtual="/inc/page-layout-top-wide.html"-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
    <head>
        <title>ListServ Archive: Dictyostelium Proteins</title>
    </head>
    <body>
        <p>
            <a name="top" id="top"></a> <a name="proteins" id="proteins"></a>
        </p>
        <h1>
            <a name="proteins" id="proteins"><b>Dictyostelium Proteins</b></a>
        </h1>
        <ul>
            <li>
                <a href="listserv_archive_proteins.html#proteins5">Has anyone successfully demonstrated release of GPI anchored proteins with phospholipase C on cells or membranes?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_proteins.html#proteins5"></a><br>
            </li>
            <li>
                <a href="listserv_archive_proteins.html#proteins4">Does anyone have an idea of the frequency of N-terminal blocking in Dicty proteins?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_proteins.html#proteins4"></a><br>
            </li>
            <li>
                <a href="listserv_archive_proteins.html#proteins3">Has anyone observed more than nine mannose residues on N-linked glycostructures on D. discoideum expressed proteins?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_proteins.html#proteins3"></a><br>
            </li>
            <li>
                <a href="listserv_archive_proteins.html#proteins2">Has anybody ever shown experimentally (e.g. by site-directed mutagenesis) what nuclear targeting signals (NLS) might look like in Dictyostelium?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_proteins.html#proteins2"></a><br>
            </li>
            <li>
                <a href="listserv_archive_proteins.html#proteins6">Protein identification: Does anyone know of a reputable facility in which mass spec and/or protein sequencing can be performed on the individual bands?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_proteins.html#proteins6"></a><br>
            </li>
            <li>
                <a href="listserv_archive_proteins.html#proteins1">What are the major proteins in a growing Dictyostelium cell? For "major" I mean protein visible on a Coomassie stained SDS page minigel.</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_proteins.html#proteins1"></a>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="proteins5" id="proteins5"></a> <b>Has anyone successfully demonstrated release of GPI anchored proteins with phospholipase C on cells or membranes, and otherwise characterized GPI-anchored plasmamembrane proteins? There are a few examples of anchored membrane proteins in Dicty i.e. CsA protein, 117 kd antigen, gp138?. However the anchors are similar to GPI anchors or somehow modified in these examples.</b><br>
            <i>-Chris Larosa, 14 Dec 1999</i><br>
        </p>
        <ul>
            <li>We have biochemically characterized a glycolipid anchor of a membrane protein gp64 of the cellular slime mold Polyspondylium pallidum (<a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=5411">Saito and Ochiai 1993</a>). The anchor is sensitive to deamination but a little resistant to phosphatidylinoditol-specific phospholipase C from Bacilus thuringiensis. This resistancy is due to the presence of amide-linked fatty acids (ceramide) and a long-chain base (phytosphingosine). This type of anchors also have been reported in Yeast. Anchored membrane proteins in Dicty has been modified a little bit, usually.<br>
                <i>-Hiroshi Ochiai and Tamao Saito, 15 Dec 1999</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>There is a paper showing release of a GPI-anchored recombinant protein from Dicty surface using an exogenous Phospholipase-D (<a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=5133">Reymond et al. 1995</a>) The authors state that, since the the GPI anchor of CsA (which was the one used in the construct) contains a ceramide in its lipid moiety (<a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=5950">Stadler et al. 1989</a>), commercially available phospholipase-C will not cleave the anchor, hence the use of PLD instead. Hope this helps.<br>
                <i>-Miguel van Bemmelen, Université de Lausanne, Switzerland, 15 Dec 99</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>After making several attempts I was not able to release our protein of interest, a glycosylated membrane protein., from plasma membranes with GPI-PLC. Of course this is a test taken from mammalian systems. What is not so clear is if GPI specific phospholipase C does anything to proteins from Dictyostelium discoideum. I have read one report where if a particular Dicty protein was reduced to the GPI anchor and a small protein fragment.... then the GPI-phospholipase C would release a fragment.<br>
                So far I have been using the commercially available GPI-PLC system, with no positive results for dicty proteins. Further I have tried freshly extracted human serum (my own blood in the cause of science!!) to try to treat membrane fractions with a serum reported to have a gpi-phospholipase D activity. Any pointers to hints or references in this area would be greatly appreciated.<br>
                <i>-P.C. LaRosa, July 2000</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Chris, It's been a long while since I've thought about GPI-PLCs, so my advice might be severely dated. Here goes anyway... My recollection is that PI-PLCs from Bacillus species (e.g., B. thuringensis) are most effective at liberating GPI-anchored proteins from intact cell surfaces. I would start with one of these.<br>
                <i>-Dale Hereld, 12 Jul 2000</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>I believe that Dicty GPI anchors, like yeast, have a ceramide rather than phospholipid tail rendering them resistant to phospholipase C. Yeast starts out with a phopholipid which then exchanges. If you can tolerate a non-enzymatic method, butanol extraction is a good selective method to isolate GPI-anchored proteins.<br>
                <i>-Chris West, U. Florida, 12 Jul 2000</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Dicty GPI anchors are indeed modified compared to other organisms. We found no PI-PLCs able to cleave a GPI anchors, except for the Dictyostelium enzyme itself. A possible way is to cleave with PI-PLD (Boehringer) and this works. Have a look at our paper: <a href="http://dictybase.org/db/cgi-bin/dictyBase/reference/reference.pl?refNo=5133">Reymond et al 1995</a>.<br>
                <i>-Christophe Reymond, Institut de biologie cellulaire et de morphologie, Lausanne, Switzerland, 13 Jul 2000</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Roche Diagnostics (formerly Boehringer) no longer sells any phospholipase D's. The JBC paper does not indicate the organism source, catalog number or specificity. Back in 1994 Boehringer did sell 3 phospholipase Ds, one from cabbage, a crude prep from streptomyces and a more pure prep from Streptomyces. Can anyone comment on which type of phospholipase D will cleave GPI anchors in Dicytostelium??<br>
                <i>-Chris LaRosa</i><br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="proteins4" id="proteins4"></a> <b>Does anyone have an idea of the frequency of N-terminal blocking in Dicty proteins?</b><br>
            <i>-Wolfgang Nellen, 30 Nov 1999</i><br>
        </p>
        <ul>
            <li>From the response I got to the question on N-terminally blocked peptides I get the impression that Dicty is, just like plants, pretty terrible. Except for Charlie Rutherford (16:24) and Richerd Gomer (5:6) all others have had more peptides blocked than with accessible N-terminus. A rough estimate would be 60 to 70% blocked, suggesting that for Edman degradation chances to get sequence without trypsination or CNBr are pretty bad.<br>
                <i>-Wolfgang Nellen, Kassel University, Germany, 3 Dec 1999</i><br>
                <br>
            </li>
            <li>I wonder if Rutherford and Gomer have better protocols to prevent in vitro blocking.<br>
                <i>-Chris Larosa, Kassel University, Germany, 3 Dec 1999</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Chris, We don't have any specific protocols other than those in general use to prevent blocking. We do use reducing reagents in the sample and electrophoresis buffer, pre-electrophoresis of gels to remove potential reactants, degassing of gel solutions to reduce the amount of catalyst needed and hence the concentration of free radicals, use sucrose instead of urea in a sample buffer, heating samples to 37C for 10 minutes, and use only sequencing grade reagents. Probably our biggest advantage is that we had our own protein sequencer in the lab so that we could take a sample directly off a PVDF membrane and load the sequencer. Regards Charlie<br>
                <i>-Charles Rutherford, Virginia Tech University, 4 Dec 1999</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Dear Dictyologists: As implied by some of the comments on N-terminal blocking, we too worry that this may be induced during purification. Though we take most of the precautions desribed by Charlie Rutherford, we have at times obtained N-terminal sequence from intact Skp1 (a cytoplasmic protein), albeit with a ragged end, and at other times nothing except after endo-lys-C. For SP85/PsA (a secretory protein), we obtained nothing from the intact protein but got the predicted N-terminal sequence after cleaving with CNBr in TFA; same thing for SP65 (could the acid have deblocked??). It seems to us that something may be going on during the purification. That's our 2-cent's (dollar's?) worth. -Chris<br>
                <i>-Chris West, University of Florida, 5 Dec 1999</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Except for having a sequencer in the lab, we do things pretty similar to Charlie Rutherford. To deal with oxidants, we put 100 microliters of mercaptoacetic acid (thioglycolic acid) into1 liter of the top electrode chamber buffer. It's like a charged beta mercaptoethanol, running at the dye front to reduce oxidants in the gel. I don't know if this affects N-termini, but it is supposed to prevent free radicals from breaking tryptophan rings.<br>
                <i>-Richard Gomer, Rice University, Houston, TX, 6 Dec 1999</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>For those who have blocked N termini, how about tryptic digestion followed by MALDI mass mapping? As the cDNA/genome sequencing progresses, this could become a much simpler way to identify proteins. Keith Williams is the Dicty expert on this, and his advice could be quite useful.<br>
                <i>-Jeff Segall, Albert Einstein College of Medicine, Bronx, NY, 6 Dec 1999</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Dear Sequencers--I'm not up to date on in vitro blocking reactions, but it is my impression that, at least in animal cells, about half of the N-termini of cytoplasmic proteins (including integral membrane proteins) are acetylated by a specific enzyme for mysterious but physiological reasons. So, this frustrating complication might not be preventable.<br>
                <i>-Ted Steck, University of Chicago, 6 Dec 1999</i><br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="proteins3" id="proteins3"></a> <b>Has anyone observed more than nine mannose residues on N-linked glycostructures on D. discoideum expressed proteins?</b><br>
            <i>-Ingrid Wilson, Macquarie University Australia, 13 May 1999</i><br>
        </p>
        <ul>
            <li>In regard to the glyco structures--I think Ellen Henderson felt that some structures were Man9. Structural proof was not there. It was done by gel filtration. Rosiland Kornfeld and Dave Sharkey found some truncated structures and I think that Sue Amathyakul found a spectrum of sizes again by gel filtration. Hope this helps.<br>
                <i>-Hudson Freeze, The Burnham Institute, 13 May 1999</i><br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="proteins2" id="proteins2"></a> <b>Has anybody ever shown experimentally (e.g. by site-directed mutagenesis) what nuclear targeting signals (NLS) might look like in Dictyostelium?</b><br>
            <i>-Thomas Winckler, Frankfurt, Germany, 26 Oct 1999</i><br>
        </p>
        <ul>
            <li>A PROSITE search predicts a bipartite NLS in the kinesin related protein K2 (KRNNPTTNIRPTVKKKLD). Using GFP fusions we have confirmed that there is an NLS in this region and narrowed it down to a small region containing the predicted NLS. We haven't mapped it to the single nucleotide level, however. K2 sequence is available here: <a target="new" href="http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&amp;db=Protein&amp;list_uids=3133267&amp;dopt=GenPept">http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&amp;db=Protein&amp;list_ui ds=3133267&amp;dopt=GenPept</a><br>
                <i>-Eugenio L. de Hostos, University of California, San Francisco, 26 Oct 1999</i><br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="proteins6" id="proteins6"></a> <b>Protein identification: Does anyone know of a reputable facility in which mass spec and/or protein sequencing can be performed on the individual bands? I would like to obtain a partial sequence of the protein so that oligos can be made. What is the best database available to search for identification of the proteins once the mass spec or sequencing data is available? Thanks,</b><br>
            <i>-Ed Harris, Jim Cardelli's laboratory, 12 Sep 2000</i><br>
        </p>
        <ul>
            <li>This site has what you need:<a target="new" href="http://www.expasy.ch/tools/#proteome">http://www.expasy.ch/tools/#proteome</a><br>
                <i>- Eugenio L. de Hostos, Exelixis Inc. South San Francisco, 12 Sep 2000</i><br>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Richard Cook in the immunology dept at Baylor does great work for us using mass spec to sequence tryptic fragments of bands and spots cut off a protein gel. They take the spot, do the trypsin digestion, and give you back sequence of several peptides. Remember to add ~0.01% thioglycollic acid to the upper chamber of the gel to prevent protein damage.<br>
                <i>-Richard Gomer, Rice University, Houston, TX, 14 Sep 2000</i><br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="proteins1" id="proteins1"></a> <b>What are the major proteins in a growing Dictyostelium cell? For "major" I mean protein visible on a Coomassie stained SDS page minigel. How many of these are cloned and sequenced? The most obvious are of course actin, myosin and discoidin. What about glycolytic enzymes or other housekeeping proteins?</b><br>
            <i>-Piero Morandini, University of Milan, Italy, 13 May 1999</i><br>
        </p>
        <ul>
            <li>There is a 2D gel giving major identified proteins available on the web at <a target="new" href="http://www.expasy.ch/ch2dothergifs/publi/dicty.gif">http://www.expasy.ch/ch2dothergifs/publi/dicty.gif</a> This is a rather old, being the immage from Jun X. Yan, Luisa Tonella, Jean-Charles Sanchez, Marc R. Wilkins, Nicolle H. Packer, Andrew A. Gooley, Denis F. Hochstrasser, Keith L. Williams. The Dictyostelium discoideum proteome - the SWISS-2DPAGE database of the multicellular aggregate (slug). Electrophoresis (1997) 18, 491-497. For more up to date information I suggest you contact the Australian Proteome Analysis Facility <a target="new" href="http://www.proteome.org.au/">http://www.proteome.org.au</a>. Cheers,<br>
                <i>-Martin Slade, Macquarie University, Australia, 13 May 1999</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p><!--#include virtual="/inc/page-layout-bottom.html"-->
        <!--#include virtual="/inc/page-footer.html"-->
    </body>
</html>
