!--#include virtual="/inc/page-header.html"-->
<!--#include virtual="/inc/page-layout-top-wide.html"-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
    <head>
        <title>ListServ Archive: Gene Disruption</title>
    </head>
    <body>
        <p>
            <a name="top" id="top"></a> <a name="ko" id="ko"></a>
        </p>
        <h1>
            <a name="ko" id="ko"><b>Gene Disruption</b></a>
        </h1>
        <ul>
            <li>
                <a href="listserv_archive_ko.html#ko6">I am trying to find out the lengths of recombinogenic arms used for generating Dicty knockouts successfully.</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_ko.html#ko6"></a><br>
            </li>
            <li>
                <a href="listserv_archive_ko.html#ko4">What are the lengths of recombinogenic arms used in successful KO experiments or what is the shortest reasonable arm length?</a>
            </li>
            <li>
                <a href="listserv_archive_ko.html#ko7">Need help figuring out suitable KO controls</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_ko.html#ko7"></a><br>
            </li>
            <li>
                <a href="listserv_archive_ko.html#ko1">PCR-based gene disruption: Has anyone tried a gene knock out in Dicty using a 60bp oligo nucleotide primer similar to what they use in the yeast system?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_ko.html#ko1"></a><br>
            </li>
            <li>
                <a href="listserv_archive_ko.html#ko2">PCR-based gene disruption: Has anyone working on Dicty has tried making gene knockouts directly with PCR products, as yeast workers now do?</a>
            </li>
            <li style="list-style: none">
                <a href="listserv_archive_ko.html#ko2"></a><br>
            </li>
            <li>
                <a href="listserv_archive_ko.html#ko5">In two cases we have now seen (by PCR and Southern blot) that a knock-out strain contains both the wild type band and the knock-out band.</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>
                <a href="listserv_archive_ko.html#ko3">While characterizing REMI mutants (made in DH1 cells), we often observed that the insertion of the plasmid (pUCBsrDBam) was accompanied by a deletion of some genomic DNA. Did this already happen to anybody ?</a>
            </li>
            <li style="list-style: none">
                <br>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="ko6" id="ko6"></a> <b>I am trying to find out the lengths of recombinogenic arms used for generating Dicty knockouts successfully. These will be placed flanking either side of the Blasticidin resistance gene and used for transformation. Recombinogenic arms of 0.3kb length, upstream and downstream of the Blasticidin gene when used for knockout did not give me knockouts, as confirmed by PCR.</b><br>
            <i>-Bhadresh Rami, Stanford University, Aug 9, 2006</i><br>
        </p>
        <ul>
            <li>I have made a couple of independent knockouts myself, and have used arms of between 800 and 950bp in length on each side of the Bsr gene. I screened approximately 100 transformants, and recovered only 4 or 5 that were targeted to the correct locus. I am not sure that having 300bp arms would give you enough targeting specificity. How many clones did you screen by PCR??<br>
                <i>-Tony Kowal, Northwestern University</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Increasing the length of the arms on both sides of the blasticidin cassette can help to increase the KO efficiency. In our laboratory, we use about 1 kb on each side of the selection cassette. However, in some cases even with an appropriate KO construct, the recombination efficiency can be very low (could be even less than 1/1000). In the past, we developed a technique to simplify PCR genotyping that allows easy testing of a large number of clones (up to 100 and more in few hours). This technique is described in: <a href="/db/cgi-bin/dictyBase/reference/reference.pl?refNo=9928">Charette S.J., Cosson P. (2004)</a>. We also developed a strategy to isolate low frequency KO clones that allow identification of rare KO mutant with frequency as low as 1/1000. This is described in:<br>
                <a href="/db/cgi-bin/dictyBase/reference/reference.pl?refNo=11189">Charette S.J. Cornillon S. and Cosson P. (2006)</a>.<br>
                <i>-Steve Charette, Université de Genève</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>There's no simple formula. We've knocked out Ras genes (which are about 500bp) by inserting a marker into the middle of a cDNA - wasn't hugely efficient, but it worked straightforwardly. Also, I knocked out something (lagC?) with a REMI rescue which had one arm less than 200bp long, though the other was quite big. Other knockouts with much bigger constructs have been unsuccessful or much less effective. I suspect the efficiency of the transfection is important, and there are gene-specific effects too.<br>
                <i>-Robert Insall, The University of Birmingham</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>I have tried to insert blasticidin cassettes at many different sites and can come to no simple conclusion except that it is unpredictable and complicated. I usually use arms on each side of the cassette of 1.0 - 1.5 kb; at the 3' end of the NsfA site, the rate of homologous recombination (HR) is ~95%. At the similar ends of the Sec 1 and DymA genes it is ~25% and below 5% respectively. At the 5' end of DymA it is also less than 5%, yet as reported by Dietmar Manstein, and I confirm, it is about 30% in the middle of the gene (for a KO). The rate of HR can therefore vary across a site and appears not to be a regional property. Yet, I have transplanted the entire Sec1 gene from it's normal locus to the 3' region of NsfA and in this new location the HR to insertion of a blasticidin cassette at the 3'end remains ~25%, and not the 95% which I hoped for. The rate of HR is unpredictable and complicated!<br>
                <i>-Mark Bretscher, MRC Laboratory of Molecular Biology</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>0.5kb on either side works if the wind is blowing in the right direction and you screen enough clones. 0.6kb or more and it works- diminishing extra returns with increasing length after this. Only ever got a KO with an arm of less than 0.5kb if the other arm is long- one long arm alone will work quite well in some cases.<br>
                <i>-Jonathan Chubb, University of Dundee</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>The discussion on arm lengths has been most interesting. Can anyone say if having upstream or downstream (or intronic?) noncoding sequence in the arms makes a difference?<br>
                <i>-Paul Fisher. La Trobe University</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>The other variable is the "type of sequence." The more unique the sequence is, the better it works. Extrageneic sequence has low sequence complexity and does not work nearly as well (in most cases) as coding sequence (however, extrageneic sequence is required for gene replacements and KOing out small genes). Similarly, when using intrageneic sequences, try to stay away from an intron being near the end of an arm or highly repeated amino acid runs. We normally shoot from 0.5+ per arm but shorter does work although with lower frequency. As frequency varies from gene to gene, the final parameter is the gene itself.<br>
                <i>-Rick Firtel, UCSD</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>I suggest checking first whether the phenotype of target gene in yeast (<a terget="new" href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>) is lethal or slow growth. If it is, no matter how long the flanking sequence is, you can't screen out the positive colonies. Does anyone have the idea to knock out a slow growth gene in Dicty?<br>
                <i>-Xinhua Liao, NIH</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>To knock out a slow growth gene in Dicty, perform the initial knockout in an axenic diploid cell line (<a href="/db/cgi-bin/dictyBase/reference/reference.pl?refNo=9440">King and Insall 2003</a>; <a href="/db/cgi-bin/dictyBase/reference/reference.pl?refNo=9531">Blagg et al. 2003</a>).<br>
                <i>-Robert Insall, The University of Birmingham</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>When HR does not work we typically try again using Bal31-exonuclease to pretreat the KO DNA to remove terminal non-homologous sequences from the cloning. Often the new attempt works, but these have been uncontrolled experiments and success may be due to chance. Maybe somebody knows if short non-homologous ends might interfere.<br>
                <i>-Chris West, University of Oklahoma</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Kindly inform what would happen if the gene of interest (the target gene) is less than 1 kb long.<br>
                <i>-Chirag Mandavia, Hunter College</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>I am not sure if non coding sequence really has a great impact in general ..because for the knockout vector that i constructed had a part of upstream non coding sequence in the 5' arm and the transformation efficiency was 60%..however it was a unique gene. In an another vector i had only the coding region and i had like 50% transformation efficiency (this gene wasn't unique).... i don't know if these results give any clue..but i beleive transformation efficiency depends on many factors such as...<br>
                - uniqueness of the gene and region of sequence chosen as the arms of the ko vector<br>
                - transformation protocol and handling of the cells<br>
                - most importantly cut the fragment that has only the 5' arm, reporter casette and 3' arm before transformation (don't use the whole linearised vector)<br>
                may be this was one of the reason that the transformation efficiency was high in my hands.<br>
                regards, Deen<br>
                <i>-Deenadayalan Bakthavatsalam, Rice University</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="ko4" id="ko4"></a> <b>What are the lengths of recombinogenic arms used in successful KO experiments or what is the shortest reasonable arm length?</b><br>
            <i>-Petr Folk, Charles University Prague, Czech Republic, 11 Oct 1999</i><br>
        </p>
        <ul>
            <li>We have managed with a cDNA of approximately 0.7 kbp. We've also failed a good few times with pieces of that size.<br>
                <i>-Robert Insall, The University of Birmingham, UK, 11 Oct 1999</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>It depends upon the gene but using DNA sequences that are not simple sequence and not overly AT rich, you can go to 0.2 kb per side. However, if you can, you should try being at 0.3 kb. The efficiency appears to be a function of the specific gene as well as length.<br>
                <i>-Rick Firtel, UCSD 11 Oct 1999</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="ko7" id="ko7"></a> <b>Need help figuring out suitable KO controls: We are trying to figure out suitable controls for making Dictyostelium knockouts. We are using the Ax2 strain of Dictyostelium. We have knocked out certain genes by homologous recombination with a copy of the gene disrupted by a blasticidin resistance cassette. On electroporation, knockouts are selected by growth in blasticidin containing medium and confirmed by PCR. Since we were looking for phenotypic differences, we needed a control apart from wild type to elminate any effects of the drug itself. Do we use an empty vector +Bsr transformed cells as control? However, these may not recombine in the genome or provide stable expression. If anyone knows of any available controls, please post your suggestions. Thanks.</b><br>
            <i>-Nitin Nair, University of Maine, Aug 8, 2006</i><br>
        </p>
        <ul>
            <li>The usual control is analyzing several independent knockout clones for every targeted gene. If you transform cells with the empty vector you might end up targeting any gene by random integration, so you might have an impredictable phenotype that you take as control. An alternative is using an extrachromosomal plasmid to have the bsr cassette in your cells, if you feel this is really needed.<br>
                <i>-Francisco Rivero, Zentrum fuer Biochemie, Cologne</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="ko1" id="ko1"></a> <b>Has anyone tried a gene knock out in Dicty using a 60bp oligo nucleotide primer similar to what they use in the yeast system? I would like to know how small a flanking region would be enough to get homologous recombination in Dicty-- And in those cases, how many transformants did you screen? In dicty it depends on the gene of interest- currently using 400bp-1kbp flanking region, we can get 5-20% of knock out strains from transformants.</b><br>
            <i>-Miho Shimizu, Ph.D. Johns Hopkins University School of Medicine, Baltimore MD, 12 Mar 2001</i><br>
        </p>
        <ul>
            <li>I did try homologous recombination with 50bp using PCR on the selection cassette and screened 100 clones. None of them appeared to be a KO. However, the standard strategy with about 500 bp gives a lot of variability (90% to less than 1% of KO), depending not only the targeted gene but also on the KO cassette.<br>
                <i>-Dr. Mohammed Benghezal, Centre Medical Universitaire, Geneve, Switzerland, 13 Mar 2001</i>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>I tried this with our Stat genes but never worked. Also Kees Weijer's group tried to KO CRAC gene which originally gives relatively high efficiency of KO with normal construct, and found that it did not work either. You could get a transformants growing but no knockout events. Although it might work in some instances, I think it would not be an alternative method against ordinary KO. Or anyone already succeeded??<br>
                <i>-Masashi Fukuzawa, Dundee University, Dundee, UK, 12 Mar 2001</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="ko2" id="ko2"></a> <b>Has anyone working on Dicty has tried making gene knockouts directly with PCR products, as yeast workers now do. In this method a marker gene is amplified by PCR with primers that have 40-50 base pair extensions with homology to the 5' and 3' regions of the gene to be disrupted. Direct introduction of the PCR product into yeast targets the marker gene to the gene to be disrupted............Has anyone tried this? Successfully? Unsuccessfully???</b><br>
            <i>- Tom Egelhoff, Case Western Reserve School of Medicine, Cleveland, OH, 29 May 1997</i><br>
        </p>
        <ul>
            <li>Some responses to my recent query didn't go to the whole Dicty group --- for those who are interested, it would seem that no one has tried the PCR-based disruption method in Dicty yet. My lab may give it a try this summer, but maybe other labs should try it too..... One reference I found for the method is [Lorenz et al, 1995, Gene 158:113-117]. Unfortunately this reference gives no technical detail on how much product is used, etc. There are a series of Medline reference for this technique in the journal YEAST, but I haven't gotten hold of a copy of this journal yet.<br>
                <i>-Tom Egelhoff, Case Western Reserve School of Medicine, Cleveland, OH, June 6 1997</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="ko5" id="ko5"></a> <b>In two cases we have now seen (by PCR and Southern blot) that a knock-out strain contains both the wild type band and the knock-out band. Surprisingly, the wild type signal in the Southern is considerably weaker than the KO signal. This is not a case of "mixed clones"! We have subcloned several times and the situation is not changed. It is not that we cant do knock-outs: other genes are cleanly disrupted without problem. Our parent strain is AX2. Has anyone else encountered that problem and found a solution? Any comments are appreciated. Send them by pm and I will compile the results for the community if you wish. Thanks a lot!</b><br>
            <i>-Wolfgang Nellen, Universität Kassel, Germany, 13 Aug 2003</i>
        </p>
        <div style="margin-left: 2em">
            <br>
        </div>
        <ul>
            <li style="list-style: none">I will try to summarize the responses I got for the "strange knock-outs":<br>
                <br>
            </li>
            <li>Many of you suggested that this may be a duplication in the genome like it has been observed for AX3. I believe this is unlikely:
                <ol>
                    <li>such duplication has not been reported for AX2
                    </li>
                    <li>because it happened in two different knock-outs in a row (hit the same duplication twice with two different genes? Murphy's law par excellence?)
                    </li>
                    <li>the signal strength in a Southern for a duplicated gene should be the same as for the knocked out gene but this was not the case.
                    </li>
                </ol>
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Some suggested a second, highly conserved copy of this gene. Again I believe this is unlikely. We have dealt with this without too much of a problem before (rrpA and rrpB differ by less than 3%). 2. and 3. from above apply
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Peter Devreotes contributed his observations with Gα, Gβ and car1 in AX3. I remember that this was tough business! But I also remember that there was the same band under the disrupted one, i.e. same signal strength. As Peter also mentioned, this was later solved by Bill Loomis (and others?) who showed the big duplication in AX2 by pulse field electrophoresis.
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Now it gets as weird as in our case! Ted Cox writes: "I don't know about solutions but Petra Fey and I found this was the rule in Polysphondylium. We often found traces of the wild type DNA elsewhere in the genome." In my reading, traces mean less than the wild type gene. Unfortunately, Ted and Petra did not solve the problem and apparently it did not hurt. But the mystery prevails!
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>If you are not interested in Polyshondylium, don't stop to worry! Barrie Coukell's observations are as bad, he writes: "I think that we have observed your strange KOs. About a year ago, we KO'd a medium size gene using blast. Our initial Southern and PCR analysis indicated that it was a single copy gene and that we had KO'd it. After maintaining the transformants for 6-8 months on 5&nbsp;µg/ml blast, we had a reason to repeat the PCR analysis. We now saw two bands, a weak one the size of the wild-type fragment and a stronger disrupted fragment. Like you, we suspected a mixed culture so we recloned our culture and also plated out spores of the original KO strain. In all cases, PCR gave the two bands described above. "<br>
                You may argue that 6-8 months continuous growth is no good but we have seen this directly after picking the first resistant clones. Barrie does not rule out that the faint wt band might have not been detected in the first Southern, especially since it was there in the cells grown from spores of the primary transformants. Unfortunately Barrie continues: "We did not repeat the Southern analysis because we lost interest in the strain. We have never explained this result. We also used (our version of) AX2." He later added another interesting observation: their gene has two transcripts and only one of them is affected in the "strange knock-out".
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>Robert Insall suggest to try the KO in diploid cells. They appear to encounter frequent aneuploidy when there is a selective disadvantage against the knock-out. We had actually thought about that but the different signal strength argues against this. Aneuploidy would just be a giant duplication. It may be worth a try anyway!
            </li>
            <li style="list-style: none">
                <br>
            </li>
            <li>My conclusion up to here is: we are not alone, at least Ted and Barry have observed similar things but they had not found a solution. In between the lines I read that Ted still got the expected KO effect while Barrie found a "partial effect" on the RNA level. We have been tinkering with several models of the disruption vector recombining in and out. This was initiated by a comment from Pierre Cosson who suggested several tandem insertions. Assuming that you start out with a duplicate vector inserting into the gene, you would get a disruption. Because of the duplication, this could recombine out and leave the wild type. If the disrupted gene was important, the KO would have a selective disadvantage but also the recombined wild type daughter that has no BsR. You may expect a dynamic population of two types of cells that are unhappy for different reasons and you would expect a growth defect that we don't see (I do not know if Barrie or Ted see this). So, there is no clear solution so far that fits with all the observations. We should maybe watch out for "strange knock-outs" and collectively find out what the problem is! Thanks a lot to all who contributed!<br>
                <i>-Wolfgang Nellen, Universität Kassel, Germany, 14 Aug 2003</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p>
        <hr>
        <p>
            <a name="ko3" id="ko3"></a> <b>While characterizing REMI mutants (made in DH1 cells), we often observed that the insertion of the plasmid (pUCBsrDBam) was accompanied by a deletion of some genomic DNA. Typically on one side of the plasmid we sequence an ORF, and on the other side non coding sequence. Sequences from the databases indicate with no ambiguity that there is a deletion, but we could never fully reconstitute the missing sequence by pasting database sequences together. Did this already happen to anybody ? What would you recommend as the next step (throw it away?)?</b><br>
            <i>-Pierre Cosson, Centre Medical Universitaire, Geneva, Switzerland, 24 Sep 1999</i>
        </p>
        <div style="margin-left: 2em">
            <br>
        </div>
        <ul>
            <li>It's not very common but not uncommon as well. The best thing to do is to take the arm with the ORF and get a cDNA by screening one of the available libraries. Then make a knockout contruct using the cDNA. You'll know from the phenotype of this whether or not the phenotype observed in the original REMI mutation is due to disruption of this ORF or possibly due to the deletion. There is also always a possibility that the original REMI phenotype is due to a mutation at a different site that arose in the original REMI mutagenesis. This will also be obvious after you make the ORF knockout. Best-<br>
                <i>- Richard A. Firtel, UCSD, CA, 24 Sept 1999</i>
            </li>
        </ul>
        <p align="right">
            <a href="#top">[TOP]</a>&nbsp;<a href="/ListServ_archive/listserv_archive_questions.html">[INDEX]</a>
        </p><!--#include virtual="/inc/page-layout-bottom.html"-->
        <!--#include virtual="/inc/page-footer.html"-->
    </body>
</html>
