<a name="top" id="top"></a>

<div class="container">
    <div class="row techniquesHead">
        <div class="techSubHeader"><h4><a class="techSubHeaderLink" href="#/research">Research</a></h4></div>
        <div class="size40"> <a href="#/research/techniques"><img src="images/logoTechniquesSmall.png" class="up"></a> Techniques </div>
        <h2>
            Parasexual Genetics in <i>Dictyostelium</i>
        </h2>
        <h6>Contributed by Jason King &amp; Robert Insall, Version 1.9 Jan 2006</h6>
        <div class="note">
            <small>
            <span class="nw">Note: </span>
            These protocols are inevitably preliminary; please tell Robert as quickly as possible if you find enhancements or different techniques
            </small>
        </div>
    </div>

    <div class="row">
        <div class="col-sm-3 col-xs-1"></div>

        <div class="col-sm-6 col-xs-10">

            <div class="blue" align="left"><h3>Contents</h3></div>
            <div class="panel panel-primary">

                <div class="list-group">
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('introduction')">Introduction</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('isolation')">Isolation of diploid strains</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('parental_strains')">Parental strains</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('maintenance')">Stability and maintenance of diploid strains</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('segregation')">Segregation</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('double')">Making double mutants</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('protocols')">Protocols</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('fusion')">Fusion of haploid cells &amp; selection of diploids</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('segregation2')">Segregation of haploid cells from diploids</a>
                    <a href="" class="list-group-item list-group-item-info" ng-click="scrollTo('cyto_stain')">Cytological staining of chromosomes to check ploidy</a>
                </div>
            </div> <!-- panel -->
        </div> <!-- col -->

        <div class="col-sm-3 col-xs-1"></div>

    </div> <!-- row -->

    <br>

    <div class="row">

        <div class="col-sm-2 col-xs-1"></div>

        <div class="col-sm-8 col-xs-10">



<!--             <p>
                <b>Download <a ng-href="#/research/techniques/word_docs/diploid_manual_RHI_1_9.doc">word file</a>.</b>
            </p> -->

            <p>
                <a name="introduction" id="introduction"></a>
            </p>
            <h3>
                Introduction
            </h3>
            <p>
                The parasexual cycle in <i>Dictyostelium</i> is fairly well understood. Two haploid cells fuse together at low frequency, leading to the generation of a diploid cell with both parents' sets of chromosomes contained within a single nucleus. These diploid cells are reasonably stable and usually divide as normal to give further diploid progeny. Occasionally the diploids can lose one copy of each chromosome and revert to a haploid state with a random mixture of chromosomes from each of the parental strains. This allows the diploid cycle to provide a means of genetic recombination. Since no meiosis is involved this process is sometimes called "parasexual recombination".
            </p>
                <br>
                <p align="center">
                    <a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a>
                </p>
            <hr>
            <p>
                <a name="isolation" id="isolation"></a> 
            </p>
            <h3>
                Isolation of diploid strains
            </h3>
            <p>
                Diploid cells form during normal growth and starvation, arising at a frequency of about once per 3x10<sup>4</sup> haploid amoebae. To isolate diploids it is therefore only necessary to select the diploid cells away from a far larger number of haploid parents. For a successful selection, a pair of strains with complementary markers is needed, so both haploid parents die under selective conditions, but a diploid formed from one cell of each parent can survive and grow. For bacterially-grown cells, two parents with complementary temperature-sensitive growth (tsg) markers have been widely used. Unfortunately, we and others have found temperature selection to be unsatisfactory for axenically growing diploids.
            </p>
            <p>
                We have been particularly successful with pairs of marked strains in which the selectable markers reside at the same genetic locus. Diploids formed from these strains are particularly stable, because when they segregate, each haploid progeny can only have one of the two markers. It is therefore essentially impossible for these strains to form a haploid which will grow under selective conditions.
            </p>

                <br>
                <p align="center">
                    <a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a>
                </p>

            <hr>
            <p>
                <a name="isolation" id="isolation"></a> 
            </p>
            <h3>
                Parental strains
            </h3>
            <p>
                We have successfully used the following strains:
            </p>
            <ul>
                <li>
                    <b>DH1:</b> AX3 cells that have a deletion in the <i>pyr56</i> locus (on chromosome 3). Normal growth requires medium containing 20&nbsp;µg/ml uracil. Selection is only efficient in FM minimal medium, though <i>pyr56</i>- cells grow far more slowly than WT in normal axenic medium. We have recently found that the problems many labs have experienced with DH1 seem to derive from the <i>pyr56</i> deletion itself: all <i>pyr56</i>- segregants from DH1-derived diploids have small colonies and ugly development. Apparently the deletion in <i>pyr56</i> also affects glycerol kinase, the next gene along. We plan to make some alternative strains soon.
                </li>
                <li style="list-style: none">
                    <br>
                </li>
                <li>
                    <b>JH8:</b> AX3-derived strain with a small deletion/frameshift in the <i>pyr56</i> gene (generated by opening a restriction site in the coding sequence, blunting it, then closing it again). Unlike DH1, develops nicely and grows well, but has the potential for reversion. Normal growth requires medium containing 20&nbsp;µg/ml uracil. As with DH1, selection is only efficient in FM minimal medium, though <i>pyr56</i>- cells grow far more slowly than WT in normal axenic medium.
                </li>
                <li style="list-style: none">
                    <br>
                </li>
                <li>
                    <b>JH10:</b> AX3-derived strain derived from JH8. Contains a deletion in the <i>pyr56</i> gene and a subsequent re-insertion of <i>pyr56</i> into the <i>thyA</i> gene, also on chromosome 3. Requires the addition of 100&nbsp;µg/ml thymidine for normal growth. Unlike DH1, these cells are efficiently selected on unsupplemented axenic medium (<i>thyA</i>- cells die in a few days without thymidine).
                </li>
                <li style="list-style: none">
                    <br>
                </li>
                <li>
                    <b>IR110:</b> AX3 cells with the <i>thyA</i> gene disrupted by G418R; haploids require thymidine for growth, and are G418 resistant. Diploids are selected with G418 without exogenous thymidine. The positive and negative selection at the same locus allows these cells to be crossed with unmarked strains.
                </li>
                <li style="list-style: none">
                    <br>
                </li>
                <li>
                    <b>IR120:</b> As IR110 except with <i>thyA</i> disrupted by hygR. Needs a fairly narrow range of hygromycin: you may need to titrate your stock to get a concentration that kills the wild types while allowing IR120 and its progeny to survive.
                </li>
                <li style="list-style: none">
                    <br>
                </li>
                <li>
                    <b>JS1:</b> As IR110 but with a cycA1 cycloheximide resistance mutation. May help in segregation (and may also get renamed to something consistent later).
                </li>
                <li style="list-style: none">
                    <br>
                    We also have some strains made in an AX2 background. Ask us if you want these. We also have some methanol-resistant versions, but have not been satisfied with methanol selection for haploids. Again, ask if you want to try.
                </li>
            </ul>

                <br>
                <p align="center">
                    <a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a>
                </p>

            <hr>
            <p>
                <a name="maintenance" id="maintenance"></a>
            </p>
            <h3>
                 Stability and maintenance of diploid strains
            </h3>
            <p>
                Diploid strains generated in this way have been relatively stable, and even under non-selective conditions only slowly revert to haploid cells. For example, after 3 weeks growth in HL-5/thymidine/uracil, ~50% of the cells in a culture of JH10/DH1 cells were still diploid. Furthermore, since we are using pairs of selectable markers which are located at the <i>thyA</i> locus, any haploids which form will only have one of the two markers. Thus even if they form, haploids will be inviable, and the diploid population can be grown for several weeks. We have found that they can usually be treated exactly like haploid cells, and in particular transformed using standard lab protocols.
            </p>

                <br>
                <p align="center">
                    <a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a>
                </p>

            <hr>
            <p>
                <a name="segregation" id="segregation"></a>
            </p>
            <h3>
                Segregation
            </h3>
            <p>
                As these diploids are relatively stable, usable levels of segregation require some method of stimulating the loss of chromosomes. It is presumed that loss of one chromosome leads to a transient aneuploid, which rapidly resolves into a true haploid - aneuploids are known to grow very poorly, and the spindles of aneuploids may also tend to lose surplus copies. We have had the most success with the microtubule-disrupting drug thiabendazole, either on SM agar as used previously or in axenic culture. Benomyl (aka benlate), which was in the past used rather more frequently than thiabendazole, is very weakly soluble in aqueous solutions and is thus tricky to use in axenic cultures.
            </p>
            <p>
                Because aneuploid cells can give some very confusing results, it is very important to make sure that segregated haploids are truly clonal. The colonies that grow up on SM agar/thiabendazole are usually not clones, but rather sectored progeny of one aneuploid cell, so segregants usually need to be recloned on a second plate before testing. If you want to be totally sure, reclone twice, but this is probably excessive for normal use. When cells are segregated in liquid medium we usually grow them for several days after thiabendazole is removed. This allows them to resolve fully into haploids, as well as recovering from the thiabendzole treatment.
            </p>
            <p>
                Segregation in axenic medium has several potential advantages. It may end up quicker, and it's possible to select particular progeny. For example, when trying to segregate a diploid containing a heterozygous knockout, you can include blasticidin in the medium. This ensures that haploids containing the wild type copy of the gene don't survive, while disruptants are OK. This can be invaluable when selecting for mutations which cause slow growth. However, we have found the proportion of haploids that arise to be much lower than when segregation is performed on SM plates.
            </p>
            <p>
                At each stage, the ploidy of haploid segregants should be checked before using them for anything involved. The simplest way is to check that they won't grow under the conditions used to select diploids (ie they're <i>thyA</i>- or <i>pyr56</i>- or neoS as appropriate). A more demanding test is to check the number of chromosomes by cytological staining with DAPI. The protocol below works well and simply so long as you have decent microscope optics.
            </p>

                <br>
                <p align="center">
                    <a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a>
                </p>

                <hr>

            <p>
                <a name="doublemutants" id="doublemutants"></a>
            </p>

            <h3>
                Making double mutants
            </h3>
            <p>
                Diploid genetics can be used to generate recombine several existing gene disruptants to make a multiple knockout. In particular, this can allow creation of multiple knockouts with only one selectable marker (usually Bsr). This only works at present if the disrupted genes are located on different chromosomes.
            </p>
            <p>
                One way of approaching this task is to make initial knockouts in a JH10/DH1 diploid background. After segregation, all progeny are then immediately ready to recombine. However, to incorporate existing mutants into diploids, one of the "universal parents" is needed. IR110 and IR120, in which the <i>thyA</i> gene is disrupted using G418R and hygR respectively, can be crossed with any neo- or hygromycin- sensitive, <i>thyA</i> wild-type strain (normal unmodified AX3 will do nicely). Diploids are generated by mixing as before, and selected in HL-5 + G418 or hygromycin in the absence of thymidine. These may then be resegregated to haploids in the presence of thymidine and G418/hygromycin, together with the marker used to generate the knockout (usually blasticidin). This will yield "universal parents" containing the desired gene disruption, which can then be crossed with further disruptant strains. This cycle can be repeated as often as needed, to give up to sextuple knockouts (one per chromosome).
            </p>
            <p>
                <strong>Two complications:</strong><br>
                (a) Once you are selecting haploids containing double or multiple mutations, blasticidin will only select away recombinants containing no disruptions at all. Single and double knockouts are equally blasticidin resistant. To separate double from single mutants, you'll need an effective screen; we usually use PCR screens which positively identify each insertion.<br>
                (b) the <i>thyA</i> gene is located on chromosome 3, so disruptants of genes located on Chr. 3 will not recombine to give "universal parents". Strains including a knockout of a gene located on Chr. 3 must therefore be added last, and haploid multiple knockouts selected without adding thymidine or G418/hygromycin.
            </p>
            <h5>
                Scheme
            </h5>
            <p>
                <img class="img-responsive" src="views/research/techniques/images/selection_scheme.png">
            </p>

                <br>
                <p align="center">
                    <a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a>
                </p>

            <hr>

            <p>
                <a name="protocols" id="protocols"></a>
            </p>
            <h3>
                Protocols
            </h3>
            <p>
                <a name="fusion" id="fusion"></a>
            </p>
            <h4>
                Fusion of haploid cells &amp; selection of diploids
            </h4>
            <ol>
                <li>
                    Mix 5 x 10<sup>6</sup> cells of each parental strain together in 10&nbsp;ml HL-5 containing any nutritional additions required (e.g. thymidine or uracil). Do not include any antibiotics for selection (G418, hyg, blast). Shake <b>slowly</b> in a flask overnight. In parallel, you should also shake control flasks containing 5 x 10<sup>6</sup> cells of each parent alone in 10&nbsp;ml of the same medium.</a>
                </li>

                <li>Label 5 Petri dishes : one diploid plate, two negative controls and two positive controls. The negative controls show that the selection is working and nothing can grow from either parent alone. The positive controls show that your selections are not too severe (a serious worry, given that the G418 and hygro knckouts are rather weakly antibiotic-resistant, and that the quality of FM medium is variable). It's worth keeping the negative controls for the duration of the experiment just to be sure that haploids can't creep through the selection. Later, when you get confident, you can omit the positive controls, but never miss out the negatives.
                </li>

                <li>After 16-24h shaking, transfer the cells to Petri dishes and leave for 1 hour to adhere. Add 0.5&nbsp;ml of the single parents to each of the positive control plates, and the remainder to the negative control plates. The diploid plate gets all 10&nbsp;ml of the flask with both parents.
                </li>

                <li>When cells have all adhered, aspirate off the old medium. Replace the medium in the diploid and negative controls with fully selective medium (ie for JH10 x DH1 diploids use FM only; for IR110 x any strain use HL-5 + 10&nbsp;µg/ml G418; for IR120 x any strain use HL-5 + 10&nbsp;µg/ml hygromycin). The positive controls should be as close to the selective medium as will allow each strain to grow:<br>
                    for JH10 use FM&nbsp;+&nbsp;thy<br>
                    for DH1 use FM&nbsp;+&nbsp;ura<br>
                    for IR110 HL5&nbsp;+&nbsp;G418&nbsp;+&nbsp;thy<br>
                    for IR120 HL5&nbsp;+&nbsp;hygro&nbsp;+&nbsp;thy<br>
                    for AX3 plain HL5).<br>
                </li>

                <li>Incubate at 22°C and feed with fresh selective medium every 2-3 days. Normally colonies take 7-8 days to appear. Typically 100-200 colonies will form in each dish and will grow at approximately wild-type rates. Make sure nothing is growing on the negative control plates.
                </li>

                <li>If diploids are not seen, check that the positive controls have all grown to confluence. This will confirm that conditions throughout the experiment are appropriate for the diploids to survive. We have found, for example, that some batches of FM lose their potency, and that different batches of hygromycin are very different in potency.
                </li>

            </ol>

                <p align="center">
                    <small><a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a></small>
                </p>

                <br>

            <p>
                <a name="segregation2" id="segregation2"></a> 
            </p>
            <h4>
                Segregation of haploid cells from diploids
            </h4>

            <div class="space1">
                <h5>
                    Segregation on bacterial lawns
                </h5>
                <p>
                    Plate out diploid cells on SM agar plates containing either 2&nbsp;µg/ml thiabendazole (from a 2&nbsp;mg/ml stock in DMSO, added to cool agar just before pouring) or 20&nbsp;µg/ml benomyl (from a 10&nbsp;mg/ml stock in DMSO; it's terribly insoluble in aqueous medium, but add it slowly while swirling the agar and disperse it well and it works). Spread 20-200 diploids, together with a suspension of <i>Klebsiella</i> in either LB or SM medium, just as you would normally do. The amount of suspension should be enough to leave the plate moist, but with no visible liquid on the surface; we usually use 200&nbsp;µl per plate. Cell growth is much slower than usual, and plating efficiency is reduced to about 30%.
                </p>
                <p>
                    After 10-14 days growth the colonies normally have faster-growing sectors as different haploid recombinants are formed. If the very edges of these sectors are picked they are often clonal and fully haploid. However, after clones are identified they should ALWAYS be re-cloned on fresh SM agar plates. As described above, colonies derive from aneuploids, so picks can contain a variety of different segregants.
                </p>

                <p align="center">
                    <small><a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a></small>
                </p>

                <h5>
                    Segregation in axenic medium
                </h5>
                <p>
                    We have been pleased with segregation in liquid medium (it may be quicker and allows a antibiotic and nutritional selections during the process), but if in doubt use bacterial plates as well. Above all, make sure your progeny are clonal. Single colonies, either from 96-well plates or on SM agar, frequently derive from an initially diploid or aneuploid cell. This means that there can be a range of different genotypes of haploid cell in the one colony.
                </p>
                <ol>
                    <li>Seed ~2 x 10<sup>7</sup> cells into a 10&nbsp;cm Petri dish in 10&nbsp;ml HL-5 medium. (NB: the precise quantities of cells and nocodazole are subject to argument at the moment; Jason reports that 2.5 x 10<sup>6</sup> cells/plate may give better efficiency).
                    </li>

                    <li>Feed with medium containing 5&nbsp;µg/ml thiabendazole and whatever selections and/or nutritional additions are required to allow the desired haploid cells to grow but cause undesirable ones to perish.
                    </li>

                    <li>Incubate at 22°C for 3 days. Growth is almost completely arrested, and many cells die in weird and wonderful ways during the course of this treatment.
                    </li>

                    <li>Remove the thiabendazole by feeding the cells twice with fresh medium, and leave to recover for 3-4 days. Make sure that all desired selections are maintained.
                    </li>

                    <li>Clone out haploid cells, either on SM agar with <i>Klebsiella</i> (the plating efficiency is somewhat variable, but usually about ~50%) or in 96-well plates. Aneuploids should usually be resolved during the recovery period, but after screening for the desired cell type it's usually advisable to reclone the strains just in case.
                    </li>
                </ol>

                <p align="center">
                    <small><a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a></small>
                </p>

                <br>

            </div> <!-- space1 -->

            <p>
                <a name="cyto_stain" id="cyto_stain"></a>
            </p>
            <h4>
                Cytological staining of chromosomes to check ploidy
            </h4>
            <ol>
                <li>
                    Seed 5x10<sup>6</sup> cells in a 5&nbsp;cm dish containing acid-washed coverslips.
                </li>

                <li>Incubate for 2 hours in 5&nbsp;ml HL-5 medium to allow cells to adhere.
                </li>

                <li>Aspirate off medium and replace with 5&nbsp;ml axenic medium + 33&nbsp;µM Nocodazole.
                </li>

                <li>Leave for 4 hours (4 seems to be best; 2 may not be enough, and 6 may make cells too sick; there's room for optimization here).
                </li>

                <li>Place coverslips in chilled distilled water for 10 minutes (again, this time could be optimized).
                </li>

                <li>Aspirate off water and fix in freshly-made, ice-cold 3:1 ethanol/glacial acetic acid for 1 hour.
                </li>

                <li>Aspirate off and re-fix in ethanol/acetic acid as before for 10 minutes.
                </li>

                <li>Remove most of the fix and air dry.
                </li>

                <li>Mount coverslips on glass slides in 3&nbsp;µl DAPI/Vectashield. The slides can be looked at immediately and can be stored in the freezer. For long storage (&gt;6 months) seal edges of coverslips with clear nail polish.
                </li>
                <li >
                    <br>
                    Nocodazole stock is 10&nbsp;mg/ml in DMSO. Pre-dissolve the stock dropwise in medium, constantly swirling before use. 1&nbsp;µl stock per ml medium = 33&nbsp;µM DAPI stock is 1&nbsp;mg/ml in water, store at -20°C. Mix 10&nbsp;µl of stock with 1&nbsp;ml Vectashield mounting medium for the correct final concentration. Store this at 4°C in the dark.
                </li>
            </ol>

                <br>
                <p align="center">
                    <a href="" ng-click="scrollTo('top')">Top</a> | <a ng-href="#/research/techniques/">Techniques</a>
                </p>

        </div> <!-- col-sm-8 -->
        <div class="col-sm-3 col-xs-1"></div>
    </div> <!-- row -->
</div>
